examples/shop-page/main.client.lua

```diff
@@ -0,0 +1,63 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Players = game:GetService("Players")
+
+local Components = ReplicatedStorage:WaitForChild("Components")
+
+local Roact = require(ReplicatedStorage.Roact)
+
+local ShopPage = require(Components:WaitForChild("ShopPage"))
+
+local screenGui = Instance.new("ScreenGui")
+screenGui.Parent = Players.LocalPlayer:WaitForChild("PlayerGui")
+
+local shopPage = Roact.createElement(ShopPage, {
+	items = {
+		{
+			identifier = "Red Visor",
```

Comment:
should we put this static data in a separate file? it may help readers focus on the mechanics of how to mount the client UI in this file

Reply:
Would you put only the items data in a separate file? Or also all the other props?

Reply:
just the items